run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+cd /usr/src/app
+case ${DOCKER_STATE} in
+create)
+    echo "running seed"
+    bundle exec rake db:create
+    bundle exec rake db:migrate
```

Comment:
It's faster and more reliable to do `rake db:schema:load` instead of running all the migrations from the bottom up.